conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/TezosGovernanceOperations.scala

```diff
@@ -230,17 +234,26 @@ object TezosGovernanceOperations extends LazyLogging {
          */
         block.data.metadata match {
           case metadata: BlockHeaderMetadata =>
-            val activeProposalAggregate = GovernanceAggregate(
-              block.data.hash,
-              metadata,
-              Some(proposal),
-              allRolls,
-              levelRolls,
-              ballotCountPerCycle,
-              ballotCountPerLevel
+            val activeProposalAggregate = currentProposal.map(
+              proposal =>
+                GovernanceAggregate(
+                  block.data.hash,
+                  metadata,
+                  Some(proposal),
+                  allRolls,
+                  levelRolls,
```

Comment:
We have a bug here. The current logic is correct for the testing_vote and promotion_vote periods where we want to show the number of rolls corresponding to `ballot` operations. However, for the proposal period, we also want to show the number of rolls corresponding to `proposal` operations. We are missing the rolls counts for proposal operations here. IIUC, the column `yay_rolls` for proposal records will always be zero.

Reply:
I'm asking because that's where all our rolls counts descend from.
We collect the responses of such endpoint for each block, without consider the voting period the block's been baked in.

Reply:
No, you are right. The logic is, in fact, correct!